### ISSUE TITLE
Add coordinate picker to identify tool (or as a separate tool)

### ISSUE THREAD

**Issue by gioman** [3liz/lizmap-web-client#3770]
Identifying on the map could (optionally) show a temporary marker where the user has clicked, show the coordinates where the user has clicked and allow the user to copy to clipboard the coordinates. Bonus allow choose the CRS for the coordinates (i.e. taking from the CRS list used in QGIS Server configs at a project level). As seen here:

![image](https://github.com/3liz/lizmap-web-client/assets/1951107/ac4b0c5a-3c4b-4f07-897b-9d3b12caf577)

**Comment by gioman** [comment]
Note: this could be exist along with an "get elevation" tool, that would return the elevation of the clicked point. making use of a public service for that.

**Comment by mfangain** [comment]
Yes, it would be great to have the possibility to copy coordinate by clicking on the map!
